src/flaresolverr.py

```diff
@@ -93,6 +94,15 @@ def controller_v1():
             logging.StreamHandler(sys.stdout)
         ]
     )
+    if log_file:
+        if not log_file.startswith("/config"):
+            log_file = "/config/" + log_file
```

Comment:
This causes issues on Windows, as you can't set an absolute or relative path.

If it's only meant for a filename, then the following works (otherwise it ends up in the C:\ root directory):

```suggestion
        if not log_file.startswith("./config"):
            log_file = "./config/" + log_file
```

But then, do we really need to be giving users the ability to name their log file? How about this instead?

```py
    log_file = os.environ.get('LOG_FILE', 'false').lower()
[...]
    if log_file == 'true':
        log_path = "./config/"
        os.makedirs(log_path, exist_ok=True)
        log_path = log_path + "flaresolverr.log"

        logging.getLogger().addHandler(logging.FileHandler(log_path))
```

Reply:
Sorry for being unclear, I'm talking about running natively on Windows, either from source or using the precompiled binaries.

Somewhat counterintuitively (for me at least), to get it to save in a subfolder of the FlareSolverr directory I had to set `log_file` as `C:/config/flaresolverr.log`

```cmd
C:\Users\Username\Downloads\FlareSolverr>set log_file=C:/config/flaresolverr.log
C:\Users\Username\Downloads\FlareSolverr>python src/flaresolverr.py
2025-10-02 01:16:48 INFO     File:C:config\flaresolverr.log
```

Actual location: `C:\Users\Username\Downloads\FlareSolverr\config\flaresolverr.log`

Other attempts:

```cmd
C:\Users\Username\Downloads\FlareSolverr>set log_file=/config/flaresolverr.log
C:\Users\Username\Downloads\FlareSolverr>python src/flaresolverr.py
2025-10-02 01:19:02 INFO     File:C:\config\flaresolverr.log
```

Actual location: `C:\config\flaresolverr.log`
(same when setting `../config/flaresolverr.log`)

```cmd
C:\Users\Username\Downloads\FlareSolverr>set log_file=./config/flaresolverr.log
C:\Users\Username\Downloads\FlareSolverr>python src/flaresolverr.py
2025-10-02 01:21:01 INFO     File:C:\config\config\flaresolverr.log
```

Actual location: `C:\config\config\flaresolverr.log`

```cmd
C:\Users\Username\Downloads\FlareSolverr>set log_file=C:\Users\Username\Downloads\FlareSolverr\config\flaresolverr.log
C:\Users\Username\Downloads\FlareSolverr>python src/flaresolverr.py
2025-10-02 01:26:26 INFO     File:C:Users\Username\Downloads\FlareSolverr\config\flaresolverr.log
```

Actual location: `C:\Users\Username\Downloads\FlareSolverr\Users\Username\Downloads\FlareSolverr\config\flaresolverr.log`

I got the same results for all of these whether using `\`, `\\`, `/` or `//`.

Not sure if running natively on Linux would have a similar issue.

Reply:
okay how about we just remove the logic and the user sets full path in ENV. 
```diff
  if log_file:
-    if not log_file.startswith("/config"):
-        log_file = "/config/" + log_file
     log_file = os.path.realpath(log_file)
     log_path = os.path.dirname(log_file)
     os.makedirs(log_path, exist_ok=True)
```

and 
```diff
    environment:
      - LOG_LEVEL=${LOG_LEVEL:-info}
-     - LOG_FILE=flaresolver.log
+     - LOG_FILE=/config/flaresolver.log
      - LOG_HTML=${LOG_HTML:-false}
      - CAPTCHA_SOLVER=${CAPTCHA_SOLVER:-none}

Reply:
Sure, looks good.

Reply:
Or:

```yml
     - LOG_FILE=${LOG_FILE:-none}
```

And then add `/config/flaresolver.log` as the README example.

We don't want people needlessly logging from a c&p of the default compose.